tests/Fake/Deep/FakeScriptInjectorContext.php

```diff
@@ -25,6 +25,7 @@ protected function doFetch($id)
 
            protected function doContains($id)
            {
+               unset($id);
```

Comment:
Fixed.

Reply:
Hey, not fixed, Sorry.

This is because to soothe phpmd. Let me hear your comment about this. Should we suppress the warning instead? unset() seems the code intentional, but "The operation doesn't contribute to the testing purpose" true.

Reply:
Looks good. But what about other tools? like PHPStorm? We suppress warning on each tool?

Reply:
Examine only src folder in phpmd which prevents unnecessary check for the sake